Apka na wersji DEV 8441 crashuje po zmianie motywu z dowolnego na dowolny

## Co powinno się dziać
Apka nie powinna crashować podczas zmiany motywu

## Co się dzieje
Apka crashuje podczas zmiany motywu z komunikatem:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.wulkanowy.dev/io.github.wulkanowy.ui.modules.main.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.appcompat.widget.ContentFrameLayout.setDecorPadding(int, int, int, int)' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3775)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3921)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5917)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5808)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:71)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ClientTransactionHandler.executeTransaction(ClientTransactionHandler.java:61)
	at android.app.ActivityThread.handleRelaunchActivityLocally(ActivityThread.java:5876)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2373)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:210)
	at android.os.Looper.loop(Looper.java:299)
	at android.app.ActivityThread.main(ActivityThread.java:8123)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:559)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.appcompat.widget.ContentFrameLayout.setDecorPadding(int, int, int, int)' on a null object reference
	at androidx.appcompat.app.AppCompatDelegateImpl.applyFixedSizeWindow(AppCompatDelegateImpl.java:1109)
	at androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor(AppCompatDelegateImpl.java:903)
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:763)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:203)
	at io.github.wulkanowy.ui.modules.main.MainActivity.onCreate(MainActivity.kt:114)
	at android.app.Activity.performCreate(Activity.java:8458)
	at android.app.Activity.performCreate(Activity.java:8427)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1385)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3756)
	... 17 more

```

## Jak to zrobić kolejny raz:
https://github.com/wulkanowy/wulkanowy/assets/43276401/393f2ad8-0eeb-49b4-b282-211398c3cf2e


## Informacje o urządzeniu i dzienniku

  - Wersja aplikacji: DEV 8441 2024-02-08
  - Wersja Androida: 13 
  - Adres URL dziennika: Nie dotyczy

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -262,7 +262,7 @@ dependencies {
     playImplementation "com.google.android.play:integrity:1.3.0"
     playImplementation 'com.google.android.play:app-update-ktx:2.1.0'
     playImplementation 'com.google.android.play:review-ktx:2.0.1'
-    playImplementation "com.google.android.ump:user-messaging-platform:2.2.0"
+    playImplementation "com.google.android.ump:user-messaging-platform:2.1.0"
 
     hmsImplementation 'com.huawei.hms:hianalytics:6.12.0.301'
     hmsImplementation 'com.huawei.agconnect:agconnect-crash:1.9.1.303'

